agent/php_rshutdown.c

```diff
@@ -109,7 +109,9 @@ int nr_php_post_deactivate(void) {
   nr_free(NRPRG(predis_ctx));
   nr_hashmap_destroy(&NRPRG(predis_commands));
 
```

Comment:
add a hashmap destroy here for the user_function_wrappers?
nr_hashmap_destroy(&NRPRG(user_function_wrappers));

this only gets called on txn start:
https://github.com/newrelic/newrelic-php-agent/pull/588/files#diff-9408f6b45682dc2cebef6e4beccd6aae78d1f1c542c82fafdb333a7d8fc83262R413